src/pages/Profiles.vue

```diff
@@ -401,6 +405,10 @@ export default class Profiles extends Vue {
         this.removingProfile = true;
     }
 
+    openDeleteProfileModal() {
+        this.$store.commit('openDeleteProfileModal', this.selectedProfile);
```

Comment:
This is an example of the poor type support of the version of Vuex we use: the second parameter is not actually supported but no warning is shown about this.

Anyway, since it looks like we end up creating a separate Vuex module for this view, I think it would make sense to store the `selectedProfile` there. One would expect to find such closely related things in the same place.